proto/reaction.proto

```diff
@@ -695,17 +695,20 @@ message ReactionAnalysis {
     SFC = 17;  // Supercritical fluid chromatography.
   }
   AnalysisType type = 1;
-  // Any details about analysis (e.g., NMR type, columns, gradients, conditions)
-  string details = 2;
+  // RSC Chemical Methods Ontology ID to define the analytical method with
+  // greater specificity. Defined at https://github.com/rsc-ontologies/rsc-cmo.
+  int32 chmo_id = 2;
```

Comment:
Should we use a `string` here and require the CHMO prefix? For example, the full ID for 1H NMR is `CHMO:0000593`.

Reply:
I'm ambivalent, but my thought was that the field name is already specific to CHMO so it isn't necessary. Do you want me to change it?

Reply:
No, int is fine; I think if we were encoding PubChem CIDs we wouldn't prefix them with `CID` either, right?

Reply:
Correct